bin/fetch-artifacts.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This module downloads test artifacts from mozilla-central continuous integration tool (Taskcluster)
+# It expects the following arguments:
+# 1) Artifacts directory:
+#    Path to the directory where the files are going to be downloaded
+# 2) Task ID:
+#    Taskcluster Task ID, identifying a precise build of Firefox
+# 3) Platform:
+#    Platform string id like linux64-opt, win32-debug, ...
+# 4) OS:
+#    Operating system string: win32, macosx64, linux32 or linux64
+
+set -e
+
+# Track each command when running on CI
+if [ $CI ]; then
+  set -x
+fi
+
+ARTIFACTS_DIR=$1
+TASK_ID=$2
+PLATFORM=$3
+OS=$4
+
+if [ ! -d $ARTIFACTS_DIR ]; then
+  echo "$0 expects a target directory as first argument"
+  exit
+fi
+if [ -z $TASK_ID ]; then
+  echo "$0 expects a Task ID as second argument"
+  exit
+fi
+if [ -z $PLATFORM ]; then
+  echo "$0 expects the Platform string as third argument"
+  exit
+fi
+if [ -z $OS ]; then
+  echo "$0 expects the OS string as fourth argument"
+  exit
+fi
+
+# On buildbot, the artifact file names aren't stable and contain the firefox version number in it
+# fetch buildbot_properties.json, which is stable, to figure out the name of artifacts
+ARTIFACT_PREFIX=target
+if [[ $OS == "win32" ]]; then
+  URL=https://queue.taskcluster.net/v1/task/$TASK_ID/artifacts/public/build/buildbot_properties.json
+  ARTIFACT_PREFIX=firefox-$(curl -L --silent $URL | grep appVersion | cut -f 4 -d '"').en-US.win32
```

Comment:
I'll followup on this curl/wget on windows. I don't think it is true anymore as I found a way to install git bash on taskcluster. And git bash comes with curl and no wget.
So I'll clean that up.
Also, fetch-artifacts.sh is used only by auto-updater for now.
It shouldn't be necessary to run it on windows.
Once we move to github, we will still be using fetch-artifacts to update the TaskID to the latest firefox nightly, but this can be done on a linux image.